pkgs/by-name/ru/rustls-ffi/package.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cargo
+, rustPlatform
+, cargo-c
+, validatePkgConfig
+, rust
+, darwin
+, curl
+, apacheHttpd
+, testers
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rustls-ffi";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "rustls";
+    repo = "rustls-ffi";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Bc9bVZ2pDsG118l/SlElZpgh9F1JEgPF8LzBX7d4mhE=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    src = finalAttrs.src;
+    name = "${finalAttrs.pname}-${finalAttrs.version}";
+    hash = "sha256-gDQ9AFrJuV7SrzKCAHQBkKj6clXuPLO0DHhnvcBqRLs=";
+  };
+
+  propagatedBuildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```

Comment:
```suggestion
  propagatedBuildInputs = lib.optionals stdenv.isDarwin [ libiconv darwin.apple_sdk.frameworks.Security ];
```

Reply:
Thanks! I missed the check failures on Darwin